assists/baremetalconfig_xlnx.py

```diff
@@ -309,7 +309,48 @@ def get_mapped_nodes(sdt, node_list, options):
     all_phandles = list(dict.fromkeys(all_phandles))
 
     valid_nodes = [node for node in node_list for handle in all_phandles if handle == node.phandle]
-    return valid_nodes
+
+    # Get Access or Domain list for the given CPU/Machine
+    root_node = sdt.tree['/']
+    root_sub_nodes = root_node.subnodes()
+    domain_nodes = []
+    for node in root_sub_nodes:
+        try:
+            is_subsystem = node["cpus"].value
+            if is_subsystem:
+                domain_nodes.append(node)
+        except:
+            pass
+
+    if domain_nodes:
+        # Get Matched Domains for a given cpu cluster
+        cpu_phandle = Lopper.node_getphandle(sdt.FDT, match_cpunodes[0].parent.number)
```

Comment:
While this is ok, there are some incoming features that further split the lopper parser and backend from libfdt, including the introducing of a dtc-free run.  The Lopper class will contain the 'always available' methods, while front/backend specific ones will move to subclasses (and in this example, one for the FDT). 

If an assist is to work with all font/backends, it shouldn't directly reference the FDT (but instead, use the LopperTree routines).

In this case, you could simply access the parent.phandle, and if it is non-zero, you know the node has a valid phandle.

Reply:
Thanks for the review... 
will fix and send the updated pull request...